scripts/perftest/python/utils_exec.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+import subprocess
+import shlex
+import re
+
+# Subprocess and log parsing related functions
+
+
+# TODO
+# return_code and better error handling
+def subprocess_exec(cmd_string, extract=None):
+    """
+    Execute the input string as subprocess
+
+    cmd_string: String
+    Input string to be executed as a sub process
+
+    extract: String
+    Based on this we extract logs accordingly
```

Comment:
You could maybe specify what are the acceptable values for the `extract` variable in the documentation.